docs/DeveloperGuide.md

```diff
@@ -260,6 +261,46 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Set/Unset payment made feature
+
+#### Implementation
+
+The purpose of the set/unset payment made feature is for tutors to modify a student's payment status.
+
+This feature implements the following operations:
+
+* `PaidCommand#execute()` - Updates the payment status of the student to paid.
```

Comment:
I was wondering if it might be better to rewrite paid as 'paid' (with quotation marks) so that it is clearer that the status is being set to 'paid' and not read as a continuous sentence?